blockchain/state_manager/src/lib.rs

```diff
@@ -1384,7 +1382,7 @@ where
         height: ChainEpoch,
         state_tree: &StateTree<&DB>,
     ) -> Result<TokenAmount, anyhow::Error> {
-        self.genesis_info.get_supply(height, state_tree)
+        self.genesis_info.get_circulating_supply(height, state_tree)
```

Comment:
I would have liked this to be inlined such that there is only one `get_circulating_supply` function. But this is also okay.